depends/packages/expat.mk

```diff
@@ -16,7 +16,7 @@ define $(package)_set_vars
 endef
 
 define $(package)_config_cmds
-  $($(package)_autoconf)
+  $($(package)_autoconf)|| cat config.log
```

Comment:
Why this change needs for other files as well?

Reply:
this was a change for debugging only when the configure step was failing. I'll remove it.